Formula/flux.rb

```diff
@@ -28,16 +28,30 @@ class Flux < Formula
     depends_on "pkg-config" => :build
   end
 
+  # NOTE: The version here is specified in the go.mod of influxdb.
+  # If you're upgrading to a newer influxdb version, check to see if this needs upgraded too.
```

Comment:
influxdb?

Reply:
This one is used in all Influxdata formulae, I think it should just be a formula at this point

Reply:
could it be `influxdb-pkg-config`?

Reply:
I'd say `pkg-config-go-wrapper` or maybe just `pkg-config-wrapper`

Reply:
#84625